### PR TITLE
create mok

### PR DESCRIPTION
**Pull request by secretuser1**
okok

### CODE REVIEW

**Review by anshumanbh — COMMENTED** [review]
you are awesome!

**Comment by anshumanbh** [comment]
I am leaving you one more comment

File: mok [hunk 1]
```diff
@@ -0,0 +1 @@
+lol
```
Comment (anshumanbh) [inline mok]: olol

**Review by anshumanbh — COMMENTED** [review]
I need some changes

File: mok (hunk 1 above)
Comment (anshumanbh) [inline mok]: this is a single comment review

**Review by kubebot-user — DISMISSED** [review]
">

**Review by anshumanbh — DISMISSED** [review]
">

**Comment by fyoorer** [comment]
"><a href=google.com>sdsda</a>

**Comment by fyoorer** [comment]
"><a href=test.com>asdada</a>

**Comment by fyoorer** [comment]
"><a href=javascript://google.com/%0aalert(1)>xxxx</a>

**Comment by fyoorer** [comment]
"><a href="JavAscRipt://google.com">sdsd</a>

**Pull request closed without merging**